src/templates/post.js

```diff
@@ -186,7 +193,11 @@ const BlogPost = ({ data }) => {
         <span className={classnames('text__paragraph__bold__grayTwo', styles.sharePosition)}>Share:</span>
         <SocialElement className={classnames(styles.socialBottom, styles.blogIcons)} links={shareXlProfileLinks} />
       </div>
-      <MoreBlogsSection data={data} refMoreFrom={refMoreFrom} title={categoryBlog.displayName} />
+      <div className={styles.newsletterAndMorBlogs}>
+        <MoreBlogsSection data={data} refMoreFrom={refMoreFrom} title={categoryBlog.displayName} />
+      </div>
+      <form onSubmit={_handleSubmit(email)}>
```

Comment:
What's this empty form?

Reply:
I need some help with the newsletter, because I followed the [plugin](https://www.gatsbyjs.com/plugins/gatsby-plugin-mailchimp/#gatsby-config-instructions) steps, but I think that my code is not very correct. And, I searched in internet, but the answers are very similar to the Gatsby page.